keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -263,37 +263,36 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  Rollouts should have no effect, as this only relates to scaledown of replicasets
 
 * **What specific metrics should inform a rollback?**
+N/A
```

Comment:
If there were metrics for number of pods deleted, maybe those could be used. Like, if there's a problem on the implementation, potentially you could see an uptick of pod deletions.

Another thing could be that the availability of applications is affected. Although the intent of the feature is that availability increases (because of better spreading).